src/main/apiClient.ts

```diff
@@ -435,7 +442,8 @@ export default class ApiClient {
               redirectUrl,
               scope: this.resolveScope(auth),
               acceptTos,
-              data
+              data,
+              returnToAfterConfirmEmail,
```

Comment:
Remove trailing comma.

Reply:
In the absence of team consensus, I propose to allow them, since we allow them on the backend. I propose to discuss it during our next tech meeting.

Reply:
Why should we allow them while there aren't any others in this project? If you want to change the convention do it in a specific PR.

Reply:
It's not a problem if there is a mix of both in one project IMO (won't change code readability).